### PR TITLE
added H2020 HARMLESS and H2020 SABYDOMA use case materials

### PR DESCRIPTION
**Pull request by vedina**
ERM for https://www.harmless-project.eu/  use cases materials

### CODE REVIEW

File: registry [hunk 1]
```diff
@@ -524,3 +524,46 @@ erm:ERM00000506 a obo:CHEBI_59999 ; rdfs:label "HARMLESS use case material I1" ;
 erm:ERM00000507 a obo:CHEBI_59999 ; rdfs:label "HARMLESS use case material I2" ; dct:creator wd:Q27061849 .
 erm:ERM00000508 a obo:CHEBI_59999 ; rdfs:label "HARMLESS use case material I3" ; dct:creator wd:Q27061849 .
 erm:ERM00000509 a obo:CHEBI_59999 ; rdfs:label "HARMLESS use case material 14" ; dct:creator wd:Q27061849 .
+
+
+erm:ERM00000510 a obo:CHEBI_59999 ; rdfs:label "SABYDOMA use case 2 material 1" ; dct:creator wd:Q27061849 .
```
Comment (egonw) [inline registry:529]: @vedina, I updated the title of the issue. Or are the labels wrong?

File: registry [hunk 2]
```diff
@@ -524,3 +524,46 @@ erm:ERM00000506 a obo:CHEBI_59999 ; rdfs:label "HARMLESS use case material I1" ;
...
+erm:ERM00000511 a obo:CHEBI_59999 ; rdfs:label "SABYDOMA use case 2 material 2" ; dct:creator wd:Q27061849 .
+erm:ERM00000512 a obo:CHEBI_59999 ; rdfs:label "SABYDOMA use case 2 material 3" ; dct:creator wd:Q27061849 .
+erm:ERM00000513 a obo:CHEBI_59999 ; rdfs:label "SABYDOMA use case 2 material 4" ; dct:creator wd:Q27061849 .
+erm:ERM00000514 a obo:CHEBI_59999 ; rdfs:label "SABYDOMA use case 2 material 5" ; dct:creator wd:Q27061849 .
```
Comment (egonw) [inline registry:533]: BTW, you're now using `dct:creator wd:Q27061849` (which is fine). Can you add a line at the top with a label, and maybe link to your ORCID page?

File: registry (hunk 1 above)
Comment (vedina) [reply registry:529]: no, there are both HARMLESS and SABYDOMA materials

File: registry (hunk 2 above)
Comment (vedina) [reply registry:533]: do you have an example ?
otherwise the link is https://orcid.org/0000-0002-4322-6179

Comment (egonw) [reply registry:533]: sure, something like:

```turtle
wd:wd:Q27061849 rdfs:label "Nina Jeliazkova"; foaf:page <https://orcid.org/0000-0002-4322-6179> .
```

(Possible with `orcid` prefix)

File: registry (hunk 1 above)
Comment (egonw) [reply registry:529]: Okay, clear!

File: registry (hunk 2 above)
Comment (egonw) [reply registry:533]: See [link](https://github.com/NanoCommons/identifiers/blob/master/registry#L10-L15)

Comment (vedina) [reply registry:533]: oh I see - I can replace it with the cordis link as for other projects

**Pull request merged**